Adapter.xml

```diff
@@ -0,0 +1 @@
+<mxfile userAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.186 Safari/537.36" version="8.3.7" editor="www.draw.io" type="device"><diagram id="2ef37502-e265-87fa-3c8f-0583108d701f" name="Page-1">7Vpfb+I4EP80SO1Dq8SBAI8FtrcrtRK6rnS7j4YYYq2Jc44psJ/+xrFN/pk2pel2b9WqKnhsj8fzm/GMPe0F083+L4HT+J5HhPWQF+17wayHkN9HIXwoykFTwvFAE9aCRmZQQXigP4kheoa6pRHJKgMl50zStEpc8iQhS1mhYSH4rjpsxVl11RSvSYPwsMSsSf2HRjLW1BEaFvTPhK5ju7IfjnXPAi9/rAXfJma9HgpW+Y/u3mDLy2w0i3HEdyVS8KkXTAXnUn/b7KeEKd1atel5tyd6j3ILksg2E/p6wiNmW2IlDhlMnaRKOnkwGgn/3SqRJhss1jTpBTfQ66V7+AvEfGOKfiV5qvv6pT5J9vIKM7o285YgGxEFT/i2Np/5ygvRoFjClFG1MUOGbS3qQ4GWFjRU2QR6JEJSQPlGCzPLxZ0Y0WaMrNQsDqNWLAdlRUFZwWTFE2ls1EemfYs3lCnr/kzYI1FcoSOWG6YGHddWC5L9SWz8I+LgSYRviBQHGLKvOsOh2twVBun3DS0uGWNoaNj4wPrIuLAD+GJMwW0WA4dZ1HSZGzmJzHZ3MZXkIcVL1buDQ6FzbaCgqo5hUx1HFZXV0Yk+hg19zBmXYMX3eE2XDd0oiy8ZljF4Y0hmmFJL3R43NIpU5yQDTdJkfZdb5CwoKH+bjeUkKfgPMuWMA+9ZwhM1U3CJJV7kSyj/SzlNZL7zwQR+QRdT73rQG4CYU2j7RRt+1XAhpzwB1pjmuBCcyR3JZM0N+h2BOqyCOh42bTxwgDroANPR72fjg/e0ccu4pJCZwDt1Un+Y95l41s1bydLKwMOgA0B9J6BznBAGSgD1MuZAN9vRDcO5tsuAKm0vY8qiO3zgWyVmJiHZsa1JzAX9CeOx9QfoFhWFqmhag7OY9KCYmWUEyWDa3KrZr5Hu8b4y8A4r/LSAsCWcZnRxtEGdmEzgsOYbM8hu8LYkT5GnvQnw/qhl8B6jDnBHjpNtsmQEJzn2F5enPNrpcg7YahmTO62qOnm/6eT9arIVw+lAkrf18XbQnsDWYtlvB6W9A70KysAFZQRefEcTcsHgT55mq7T6iyJ9QPsaaPstvbQTaF1XL3VdWlGibqsKUnlIyQegrwA0DH8hoM1Lk7okEKGzqJsIp7J9uPVdCZYbvLOjMuz6zw3BgVcNwX37RFOyhWDosAUfdZFMhye8O3JlYOYEb5OdfXh/2QIGT3o/anq/E/FOvL/5RNDALg/XFjv7ruifBVz9KvVVATm78tterurIMbwgbM4zKilX/IUeW0P0V4A2QO1AG3WAmesJQDlpRqTVxcVe+6baKxixEtO6K5Auj41HDlyVIQW+fvYsfbV+/5Vf7MuzX8bw4yA426bCZl7ntKlOnlHGzx8EJIluVJVCxXaGs4wuq09JZE/lN6VK0JFufTc9Ec7i44MUbF8cvhmV543vx0l6TRI16hy1+Aly8a1YmlEmJ4UcYU3s4XhCtSVdDp54oROEYUkfq1K49GtWmPPcMWwUr92jkVdDSAtvJqFSnaPGBw0HVUb1M19vucEIYMKH0jBj2CflHdXk9SvlF/iiGRaWdNRoK+NCzTe6ovChP7cZKdVE3HWQP/MBr4Ok0feef7cZvVEG4Sy6vFstbop6NxOqOlfqwRuakzYFui+mLvJUia45vVq007RY5cWZrQxb+fzTZcOzlKTD4VFLTd4vi93NDTbTiaeZWB7gT4nlclnSU5n+lPoahOe0Xjsi/l+lUtQPrqsHvB86SkmjwJFHdhDzkeu298KYXy8/FDnA2CtnAd61h2x7TgQFYdXZPSuSxJcGfftwUY76o/cM+rXCd1hPy9oG/bDGx6/xORHzz4nLLW5/zxlAJbU7bQ2+F1StwRudsIYiPdRppE0Q/U4t5V3Tw9/eUqBZ/DOPHl78x1Tw6T8=</diagram></mxfile>
```

Comment:
Diagram does not present the code. There is no attribute *drawPanelController* in *PloterDrawerAdapter*.